language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ModulePartNodeContext.java

```diff
@@ -142,7 +141,11 @@ protected List<LSCompletionItem> getCompletionItemsOnQualifiers(Node node, Balle
             case TRANSACTIONAL_KEYWORD:
                 completionItems.add(new SnippetCompletionItem(context, Snippet.DEF_FUNCTION.get()));
                 break;
+            case CONFIGURABLE_KEYWORD:
+                completionItems.addAll(this.getTypeDescContextItems(context));
```

Comment:
We only allow subtypes of `anydata` as the type descriptor in this context. Shall we suggest only those? @IMS94 WDYT?  Refer https://ballerina.io/spec/lang/2021R1/#anydata

Reply:
Yes, that's true. In `ModuleVariableDeclarationNodeContext` also uses this `getTypeDescContextItems` method to get the suggestions. I think we need another method to get the subtypes of `anydata`. @malinthar @IMS94